android/modules/calendar/src/java/ti/modules/titanium/calendar/EventProxy.java

```diff
@@ -126,6 +129,16 @@ public static String getExtendedPropertiesUri()
 		return events;
 	}
 
+	@Kroll.method
+	public void save() {
+		// Currently only saving added recurrenceRules.
+		String ruleToSave = ((RecurrenceRuleProxy) ((Object[]) getProperty(TiC.PROPERTY_RECURRENCE_RULES))[0]).generateRRULEString();
+		ContentValues contentValues = new ContentValues();
+		contentValues.put(Events.RRULE, ruleToSave);
```

Comment:
Not a huge fan of the truncated constant name here: `Events.RRULE` versus something like `Events.RECURRENCE_RULE`